share/completions/equery.fish

```diff
@@ -1,104 +1,123 @@
+		'$category'     "Contains just the category (e.g. 'app-portage')." \
+		'$name'         "Contains just the package name (e.g. 'gentoolkit')." \
+		'$version'      "Contains the package version (without the revision) (e.g. '0.3.0_rc10')." \
+		'$revision'     "Contains the package revision (e.g. 'r1')." \
+		'$fullversion'  "Contains the package version including its revision (e.g. '0.3.0_rc10-r1')." \
+		'$slot'         "Contains the package's slot." \
+		'$repo'         "Contains the name of the package's repository (e.g. 'gentoo')." \
+		'$mask'         "The Mask-status field (~M-??)." \
+		'$mask2'        "Contains a verbose description of the packages masking status." \
+		'$location'     "The Location field (IPO-)."
 end
 
-#########################################################################
-# Global Opts
-complete -c equery -s q -l quit -d "causes minimal output to be emitted"
-complete -c equery -s C -l nocolor -d "turns off colours"
-complete -c equery -s h -l help -d "displays a help summary"
-complete -c equery -s V -l version -d "displays the equery version"
+## Global Opts
+complete -c equery -s h -l help -d "displays a help message"
+complete -c equery -s q -l quiet -d "minimal output"
+complete -c equery -s C -l no-color -d "turns off colors"
 complete -c equery -s N -l no-pipe -d "turns off pipe detection"
-# END Global Opts
-#########################################################################
-
-#########################################################################
-# Subcommands
-complete -c equery -n '__fish_use_subcommand' -xa 'belongs\t"'(_ "list all packages owning file(s)")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'check\t"'(_ "check MD5sums and timestamps of package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'depends\t"'(_ "list all packages depending on specified package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'depgraph\t"'(_ "display a dependency tree for package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'files\t"'(_ "list files owned by package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'hasuse\t"'(_ "list all packages with specified useflag")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'list\t"'(_ "list all packages matching pattern")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'size\t"'(_ "print size of files contained in package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'uses\t"'(_ "display USE flags for package")'"'
-complete -c equery -n '__fish_use_subcommand' -xa 'which\t"'(_ "print full path to ebuild for package")'"'
-# END Subcommands
-#########################################################################
-
-#########################################################################
-# Local opts
+complete -c equery -s V -l version -d "display version information"
+
+## Subcommands
+complete -c equery -n '__fish_use_subcommand' -xa 'belongs'     -d '"'(_ 'list all packages owning file(s)')'"'
```

Comment:
Descriptions are translated by default - no need to invoke `_` at all.

Just `-d 'List all packages owning files'`.

Also:

- Capitalize the first word

- Keep descriptions short (i.e. no `file(s)`, just `files`)

Reply:
Good, that part was inherited and I left it in just in case.